src/products/mozilla_org/testing.md

```diff
@@ -52,28 +52,62 @@ You can make linguistic changes directly in [Pontoon](https://pontoon.mozilla.or
 * Test the page layout on the leading locally developed browser if available.
 * Test the page layout on mobile devices of major platforms.
 
-## When can I see the localized page on the production server?
+## Page activation
 
-Updated translations are pushed to the production server almost daily.
+Updated translations are pushed to the production server regularly throughout the day.
 
-When a brand new page is available for localization, it won’t be enabled in production until it’s fully localized. When existing pages receive updates with new strings, this new content won’t be displayed on production until localized, to avoid displaying a mix of English and localized text.
-
-In some cases, pages receive major updates that require a complete rewrite of the template. If this happens, the old template is kept online only for a short period of time and, when removed, it will cause the URL to redirect users to the English version.
+* When a brand new page is available for localization, it won’t be enabled on production until it’s at least 80% complete. This means, after it reaches this threshold, the page will be on production in English and target language.
+* When an existing page receives updates with new strings, these new strings may or may not have fallback strings. For a page that is completely translated, depending on the rule set for the page, the update may appear once fully localized, or the update will be in English.
```

Comment:
I'm completely lost here. Can you try to explain what you were trying to describe?

Do we support fallbacks like tags in lang files? What's determining the behavior?

Reply:
Bedrock would have something like a tag in place, but not for all the updates, so some updates would show English right away (those already activated locales). If there is a tag, it will wait for the tagged string localized and switch.

Reply:
I think this requires a large explanation. Can you describe this as two possible scenarios, with more details?

Reply:
I added an extra bullet: 1). update without a tag so english shows up right away with target language (activate page only) 2). update with a tag to allow completion in a defined period of time.